packages/graphql_client/lib/src/farms.dart

```diff
@@ -1,43 +1,45 @@
 import 'package:graphql_client/graphql_client.dart';
 import 'package:graphql_client/models/farms.dart';
+import 'package:graphql_client/models/public_ips.dart';
 
 class TFFarms {
   final GraphQLClient gqlClient;
 
   TFFarms(this.gqlClient);
 
-  Future<List<FarmInfo>> listFarmsByTwinId(
-      ListFarmsByTwinIdOptions options) async {
-    print(options.twinId);
+  Future<List<FarmInfo>> listFarms(FarmsQueryOptions? queryOptions,
```

Comment:
The function already returns `FarmInfo`, the result shouldn't be printed unless i do it myself